shared/src/iosMain/kotlin/ch/dreipol/multiplatform/reduxsample/shared/database/DriverFactory.kt

```diff
@@ -1,11 +1,26 @@
 package ch.dreipol.multiplatform.reduxsample.shared.database
 
 import ch.dreipol.multiplatform.reduxsample.shared.delight.Database
+import co.touchlab.sqliter.DatabaseConfiguration
 import com.squareup.sqldelight.db.SqlDriver
 import com.squareup.sqldelight.drivers.native.NativeSqliteDriver
+import com.squareup.sqldelight.drivers.native.wrapConnection
+import platform.Foundation.NSFileManager
 
 actual class DriverFactory : DriverCreator {
     actual override fun createDriver(): SqlDriver {
-        return NativeSqliteDriver(Database.Schema, "app.db")
+        val groupUrl = NSFileManager.defaultManager
+            .containerURLForSecurityApplicationGroupIdentifier("group.ch.dreipol.reZHycle")
+        val schema = Database.Schema
+        val config = DatabaseConfiguration(name = "app.db",
+            basePath = groupUrl!!.path,
```

Comment:
I assume you want the app to crash, if groupURL is null, correct? I would prefer a Excpetion with a message, to not have a forced unwrap.